qiskit/providers/fake_provider/fake_generic.py

```diff
@@ -0,0 +1,525 @@
+        self._supported_gates = get_standard_gate_name_mapping()
+
+        if coupling_map is None:
+            self._coupling_map = CouplingMap().from_full(num_qubits)
+        else:
+            if isinstance(coupling_map, CouplingMap):
+                self._coupling_map = coupling_map
+            else:
+                self._coupling_map = CouplingMap(coupling_map)
+
+            if num_qubits != self._coupling_map.size():
+                raise QiskitError(
+                    f"The number of qubits (got {num_qubits}) must match "
+                    f"the size of the provided coupling map (got {coupling_map.size()})."
+                )
+
+        self._basis_gates = basis_gates if basis_gates else ["cx", "id", "rz", "sx", "x"]
+        for name in ["reset", "delay", "measure"]:
+            if name not in self._basis_gates:
+                self._basis_gates.append(name)
+
+        self._build_generic_target()
+        self._build_default_channels()
+
+    @property
+    def target(self):
+        return self._target
+
+    @property
+    def max_circuits(self):
+        return None
+
+    @property
+    def dtm(self) -> float:
+        """Return the system time resolution of output signals
+
+        Returns:
+            The output signal timestep in seconds.
+        """
+        if self._dtm is not None:
+            # converting `dtm` in nanoseconds in configuration file to seconds
+            return self._dtm * 1e-9
+        else:
+            return None
+
+    @property
+    def meas_map(self) -> list[list[int]]:
+        return self._target.concurrent_measurements
+
+    @property
+    def noise_defaults(self) -> dict[str, tuple | None]:
+        """Noise default values/ranges for duration and error of supported
+         instructions. There are two possible formats:
+
+            #. (min_duration, max_duration, min_error, max_error),
+                if the defaults are ranges
+            #. (duration, error), if the defaults are fixed values
+
+        Returns:
+            Dictionary mapping instruction names to noise defaults
+        """
+
+        if self._noise_defaults is None:
+            # gates with known noise approximations
+            default_dict = {
+                "cx": (1e-8, 9e-7, 1e-5, 5e-3),
+                "ecr": (1e-8, 9e-7, 1e-5, 5e-3),
+                "cz": (1e-8, 9e-7, 1e-5, 5e-3),
+                "id": (3e-8, 4e-8, 9e-5, 1e-4),
+                "rz": (0.0, 0.0),
+                "sx": (1e-8, 9e-7, 1e-5, 5e-3),
+                "x": (1e-8, 9e-7, 1e-5, 5e-3),
+                "measure": (1e-8, 9e-7, 1e-5, 5e-3),
+                "delay": (None, None),
+                "reset": (None, None),
+            }
+            # add values for rest of the gates (untested)
+            for gate in self._supported_gates:
+                if gate not in default_dict:
+                    default_dict[gate] = (1e-8, 9e-7, 1e-5, 5e-3)
+            self._noise_defaults = default_dict
```

Comment:
I thought it would make sense to keep the fixed noise ranges that we already know work (tested in the follow-up PR). For the untested gates, these values are going to be a bit handwavy, but I would guess they are unlikely enough to show up not to be a big problem at the moment.